lib/diversity/engine.rb

```diff
@@ -31,6 +31,11 @@ def initialize(options = {})
       @debug_level = 0
       @logger = @options[:logger]
 
+      @cache = Moneta.build do
+        use :Expires, expires: 3600
+        adapter :Memory
```

Comment:
Fredrik Liljegren skrev den 2015-03-26 15:56:
> In lib/diversity/engine.rb
> <https://github.com/DiversityTemplating/diversity-ruby/pull/39#discussion_r27219840>:
>
>> @@ -31,6 +31,11 @@ def initialize(options = {})
>>        @debug_level = 0
>>        @logger = @options[:logger]
>>
>> +      @cache = Moneta.build do
>> +        use :Expires, expires: 3600
>> +        adapter :Memory
>
> The LRUHash isn't compatible with Expires. Expires takes the value and
> places it in an array with a timestamp, and the LRUHash craches on the
> array not having bytesize :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DiversityTemplating/diversity-ruby/pull/39/files#r27219840>.
>

Well, _someone_ (*cough*, *cough*) might already have had the same 
trouble and opened up an issue...which rightfully got closed half an 
hour later:

https://github.com/minad/moneta/issues/76